roc/log.go

```diff
@@ -182,6 +185,24 @@ func loggerRoutine() {
 	}
 }
 
+func logWrite(level LogLevel, text ...interface{}) {
+	currentLogLevel := logLevel.Load()
+	if currentLogLevel != nil {
+		if currentLogLevel := currentLogLevel.(LogLevel); level <= currentLogLevel {
+			loggerCh <- LogMessage{
+				Level: level,
+				Time:  time.Now(),
+				Pid:   uint64(os.Getpid()),
+				//Tid:    -get from C module-
+				Module: "roc_go",
+				File:   "log.go",
+				Line:   210,
```

Comment:
I guess you hard-coded file and line accidentally? Obtaining them from stack, as it was in your previous version of the patch, was a good implementation.